conans/client/installer.py

```diff
@@ -326,12 +326,14 @@ def _handle_package(self, package, install_reference, remotes, handled_count, to
 
         # Make sure that all nodes with same pref compute package_info()
         pkg_folder = package_layout.package()
+        pkg_metadata = package_layout.metadata()
         assert os.path.isdir(pkg_folder), "Pkg '%s' folder must exist: %s" % (str(pref), pkg_folder)
         for n in package.nodes:
             n.prev = pref.revision  # Make sure the prev is assigned
             conanfile = n.conanfile
             # Call the info method
             conanfile.folders.set_base_package(pkg_folder)
+            conanfile.folders.set_base_pkg_metadata(pkg_metadata)
```

Comment:
This allows deployers to access ``dep.package_metadata_folder`` to collect and deploy metadata.

TODO: Discuss if we want to drop this, and force users to automate it via custom command with ``download + cache path`` over the list of dependencies.